troposphere/static/js/components/modals/instance/InstanceStartModal.jsx

```diff
@@ -26,7 +30,33 @@ export default React.createClass({
     // ------
     //
 
-    renderBody: function() {
+    renderBody: function(multiple, instances) {
+        let operativeNoun = multiple ? "these instances": "this instance",
+            instanceList = (<div/>);
+
+        if (instances) {
+            instanceList = (
+            <div>
+                <ul>
+                {instances.map((i) => {
```

Comment:
I am leery of reaching for a mixin to eliminate the repetition since the React community seems to have eschewed (and, I believe it makes the move to `React.Component` more difficult in the future).